app/Http/Controllers/Front/BaseCartController.php

```diff
@@ -237,14 +243,12 @@ private function getCartValues($productId, $canReduceAgent = false)
             throw new \Exception('Product not present in cart.');
         }
 
-        if ($hasPermissionToModifyAgent) {
-            if ($canReduceAgent) {
-                $agtqty = $agtqty / 2;
-                $price = \Cart::getTotal() / 2;
-            } else {
-                $agtqty = $agtqty * 2;
-                $price = \Cart::getTotal() * 2;
-            }
+        if ($canReduceAgent) {
+            $agtqty = $agtqty / 2;
+            $price = \Cart::getTotal() / 2;
+        } else {
+            $agtqty = $agtqty * 2;
+            $price = \Cart::getTotal() * 2;
```

Comment:
write doc comment to explain this logic

Reply:
done

Reply:
I meant the document the logic definition why are we dividing or multiplying by 2?